_docs/4-kubernetes/1-controlplane.md

```diff
@@ -0,0 +1,74 @@
+---
+layout: page
+title: Control Plane
+permalink: /docs/kubernetes-integration-control-plane.html
+redirect_from: /docs/kubernetes-integration/controlplane
+category: Kubernetes Integration
+order: 2
+---
+
+### Deploying Amalgam8 Control Plane <a id="deploy"></a>
+
+Amalgam8 rus a control-loop to validate routing rules stored in Third Party Resources (TPR)
+ before they are fetched by the sidecars.
+ The Kubernetes controller is deployed using the ReplicationController specification found in
+ [`testing/kubernetes/controlplane.yaml`](https://github.com/amalgam8/amalgam8/blob/master/testing/kubernetes/controlplane.yaml):
+
+```yaml
+apiVersion: v1
+kind: ReplicationController
+metadata:
+  name: rules
+  labels:
+    app: rules
+spec:
+  replicas: 1
+  selector:
+    app: rules
+  template:
+    metadata:
+      name: rules
+      labels:
+        app: rules
+    spec:
+      containers:
+        - name: rules
+          image: amalgam8/a8-k8s-rules-controller:latest
+          imagePullPolicy: IfNotPresent
+          env:
+            - name: K8S_NAMESPACE
+              valueFrom:
+                fieldRef:
+                    fieldPath: metadata.namespace
+```
+
+To deploy:
+
+```bash
+$ kubectl create -f testing/kubernetes/controlplane.yaml
+```
```

Comment:
All docs assume user has downloaded a zip of examples release. So no references to testing/.. 
instead it's examples/
for alpha testers, we can tell them what to copy from testing/kubernetes into examples.